src/core/src/op/fake_convert.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/fake_convert.hpp"
+
+#include "itt.hpp"
+
+namespace ov {
+namespace op {
+namespace v13 {
+
+FakeConvert::FakeConvert(const ov::Output<ov::Node>& arg,
+                         const ov::Output<ov::Node>& scale,
+                         const ov::Output<ov::Node>& shift,
+                         const std::string& destination_type,
+                         bool apply_scale)
+    : Op({arg, scale, shift}),
+      m_destination_type(destination_type),
+      m_apply_scale(apply_scale) {
+    constructor_validate_and_infer_types();
+}
+
+bool FakeConvert::get_apply_scale() const {
+    return m_apply_scale;
+}
+
+const std::string& FakeConvert::get_destination_type() const {
+    return m_destination_type;
+}
+
+const std::vector<std::string> FakeConvert::m_valid_types({"F8E4M3", "F8E5M2"});
```

Comment:
please, avoid  global vars. Better to use lazy initialization via function returning static object 
as a reference https://github.com/openvinotoolkit/openvino/pull/19976

Reply:
> I would say it's a static member of FakeConvert class

Anyway, it's a global var :)